### ISSUE TITLE
How to understand the code of the center_normal

### ISSUE THREAD

**Issue by XipengY** [ingra14m/Specular-Gaussians#21]
Hi, thanks for your excellent work~

We have some questions, the code: https://github.com/ingra14m/Specular-Gaussians/blob/main/gaussian_renderer/__init__.py#L229
center_normal = -torch.mean(grid_offsets, dim=1)
How to understand the center_normal equals to -torch.mean(grid_offsets, dim=1) ? Paper says "Normal estimation. Following [19, 43], we use the shortest axis of each Gaussian as its normal.", why the mean of  the offset is the shortest axis?
